### ISSUE TITLE
If by some miracle we convince a bank or credit union (or tech company on top of a bank) to integrate with OC's API will OC support it?

### ISSUE THREAD

**Issue by mlncn** [opencollective/opencollective#7646]
Wise has proven to be much more difficult to work with than even a bank, and the fees are the injury on top of the insult.

## Who is your user?

[People Who Give a Damn, Inc.](https://opencollective.com/pwgd), is a fiscal host to several collectives on the Open Collective platform.

## What are they trying to achieve?

Our banking services provider is a green-oriented fintech with a traditional bank holding the money.  They send unlimited ACH and wire transfers for no fee.  In theory they would be well positioned to have their API integrate with Open Collective's and we could cut Wise out of the equation.

When we need to pay out money, we want to do so directly through our banking services provider with integration with Open Collective, so we save the trouble and expense of sending money to Wise.

## How are they currently doing this?

We are able to send money with our banking services provider, but manual marking of paid, especially since Open Collective does not support marking money as sent and then easily updating the completed date later, makes this operationally onerous.

## How well understood is this problem?

- [ ] Surfacing: if you feel this is a seed of an idea that needs to be researched.
- [ ] Understanding: if you've done some research and feel it is already well understood and ready for shaping.
- [ ] Shaping: if you feel a solution is clearly defined and can be made into a pitch ([read more about shaping](https://basecamp.com/shapeup/1.1-chapter-02))
- [x] Pitching: if you feel that it is ready to be presented for prioritisation ([read more about pitching](https://basecamp.com/shapeup/1.5-chapter-06))

What can we show the financial services provider regarding integration, and would their name be listed as an option next to PayPal and Wise?

**Comment by Betree** [comment]
Hi @mlncn, thanks for explaining your use case.

We recently started integrating with [Plaid](https://plaid.com/solutions/open-finance/). For now we're using it the other way around, to let host admins easily import their bank transactions into Open Collective (e.g. as Added Funds).

But Plaid can also be used to trigger payments from a connected bank account. Could you share with me (here or by private message on Discord, depending on how private it is) your bank name? Or check by yourself whether it is compatible with Plaid?

**Some additional questions:**

> Wise has proven to be much more difficult to work with than even a bank, and the fees are the injury on top of the insult.

Could you summarize which issues you faced with Wise, just so we can better understand? Their fees are usually considered pretty low in the industry (at least for international transfers) so I'm especially curious about that part.

> Open Collective does not support marking money as sent and then easily updating the completed date later

Could you detail this as well? What would be your expectation in terms of interface/flow?

**Comment by mlncn** [comment]
@Betree Thank you and apologies for the slow response!

The first fight with Wise was about Plaid.  Although clearly industry standard, Plaid's method of operation is unacceptable— holding onto the **username** and **password** of **your bank account**, and then asking you to pass on one time auth codes from the bank to Plaid.  This violates the terms of service of most banks and is simply horrible security practice.  Unlike other services, Wise did not allow a fallback to old-fashioned authentication of ownership of a bank account by giving the amounts of a couple tiny, temporary deposits it makes.

Turns out wiring money into Wise was cheaper than Plaid anyway, once we realized outgoing wires were miraculously free from our fintech/bank.

Which is a long way of saying we would not want to encourage more people to use Plaid.  Regular old ACH still actually works, despite the pathetic state of the US banking system.

The main problem with Wise is that it keeps requesting more and more documentation.  We gave it initial documentation, and then later it asked for business’ registration document, and while scarily described we were able to provide that.  And things would seem fine and then we would try to transfer money *into* Wise and it would then ask for yet more documentation, and send the money back if we did not provide it quickly enough.  What stopped us completely, for now, is a requirement (again, not provided up front) to give the date of birth of all of our board members.  We have a large board that is in the process of being re-invigorated and could not get the birthdate for everybody, certainly not within the initial 7 days, and at this rate we expect Wise will turn around and ask for social security numbers or blood samples next, what with the constantly moving goalposts.

> We're getting in touch about your recent transfer set up through the Open Collective application via Wise.
> Due to our regulatory obligations as a financial institution, we're sometimes required to conduct additional due diligence. This means that we’re required to ask for additional information from our customers.
> 
> Before we continue with the transfer, we kindly ask you to provide the following information:
>
>  - Full name, date of birth and country of residence of all members who control the organisation.
>  - Role of [volunteer already freaked out by Wise] in the organisation
>
> If we don’t hear back from you in the next 7 business days, we'll refund any money paid into your Wise account – we can’t hold your money for too long.
> 
> If you need any further assistance, please let us know – we're always happy to help.
> 
> Regards,
> The Wise Team

So anyway for the moment we have been sending the money directly from our bank, which led to the two questions:

1.  If by some miracle the fintech/bank is open to providing an API for creating contacts and initiating ACH payments, would the Open Collective platform be willing to work with that API?
2.  If not / in the meantime, when we "Go to pay" in the interface, and choose "Manual", we should be able to leave the optional **Effective Date** blank to indicate that the transfer has been initiated but not completed, and then when we see it completed in the bank we set the Effective Date to mark it as complete.  Currently we cannot edit the manual transfer without revoking it entirely, which we certainly do not want to do.

**Comment by mlncn** [comment]
Well i asked the fintech/bank and, at least according to frontline support, they have no plans to provide any such API.

Which leaves us getting things back to working with Wise.

OC can still improve manual transfers but that's a different issue.

**Comment by Betree** [comment]
> The main problem with Wise is that it keeps requesting more and more documentation.

From what we've seen so far around diverse groups and payment providers, this can't really be avoided, and Wise is not the worst. Some providers like Alipay refuse to serve anything related to non-profits or donations. Others, like PayPal, will not let you access features like "PayPal Payouts" based on arbitrary conditions and heavy documentation requests.
The frustrating part is often that their requirements, though understandable for large organizations, seem overkill for smaller ones that are just interested in moving a few thousand dollars per year.

But there again, there's not much we can do at this stage. We would love to see a better banking standard API emerge in the US or globally, but it's not there yet. Plaid is the closest thing we see to this solution, and we could decide to invest more in it in the future. In the meantime, Wise is, in our experience, the simplest and cheapest option for sending money worldwide as a fiscal host.

Closing this discussion as it seems everything was answered. Feel free to re-open if needed.
